src/github.com/getlantern/lantern-ui/app/partials/error.html

```diff
@@ -0,0 +1,46 @@
+<div id="error" ng-controller="ErrorCtrl" ng-show="showError">
+  <div id="oops">
+    {{ 'ERROR_OOPS' | translate }}
+  </div>
+  <div id="error-description">
+    {{ 'ERROR_DESCRIPTION' | translate }}
+  </div>
+  <div id="error-reasons">
+    <ol>
+      <li ng-show="!model.settings.systemProxy">
+        <div>{{ 'PROXY_OFF_ISSUE' | translate }}</div>
+        <div ng-show="!showExtensionHelp"><a ng-click="toggleShowExtensionHelp()">{{ 'WHAT_DO' | translate }}</a></div>
+        <div class="error-action ng-hide" ng-show="showExtensionHelp">
+          {{ 'PROXY_OFF_HELP' | translate }}
+        </div>
+      </li>
+      <li>
+        <div>{{ 'BROWSER_EXTENSION_ISSUE' | translate }}</div>
+        <div ng-show="!showExtensionHelp"><a ng-click="toggleShowExtensionHelp()">{{ 'WHAT_DO' | translate }}</a></div>
+        <div class="error-action ng-hide" ng-show="showExtensionHelp">
+          {{ 'EXTENSION_HELP' | translate }}
+        </div>
+      </li>
+      <li>
+        <div>{{ 'XUNLEI_ISSUE' | translate }}</div>
+        <div ng-show="!showXunleiHelp"><a ng-click="toggleShowXunleiHelp()">{{ 'WHAT_DO' | translate }}</a></div>
+        <div class="error-action ng-hide" ng-show="showXunleiHelp">
+          {{ 'XUNLEI_HELP' | translate }}
+        </div>
+      </li>
+      <li>
+        <div>{{ 'ANOTHER_ISSUE' | translate }}</div>
+        <div ng-show="!showConnectionHelp"><a ng-click="toggleShowConnectionHelp()">{{ 'WHAT_DO' | translate }}</a></div>
+        <div class="error-action ng-hide" ng-show="showConnectionHelp">
+          <div>{{ 'CONNECTION_HELP' | translate }}</div>
+          <div>Windows Vista/7/8/10: C:\Users\USERNAME\AppData\Roaming\Lantern\logs</div>
```

Comment:
@myleshorton I don't quite understand. Do you mean to set up different link for each translation?